docs/developer-docs/backend/rust/8-optimizing.md

```diff
@@ -0,0 +1,86 @@
+# 8: Optimizing Rust canisters
+
+## Overview
+Compiling Rust to Wasm often increases the file size significantly. dfx versions 0.14.0 and new include a new `wasm-opt` optimization package that can be used to optimize cycle consumption and binary size. 
```

Comment:
This section and some parts below are written as though we have created wasm-opt. It should probably be clarified that what we have created is ic-wasm, one of the features of which is wasm optimization using the existing wasm-opt package from binaryen.